Team6220_2020/src/main/java/org/firstinspires/ftc/team6220_2020/Constants.java

```diff
@@ -0,0 +1,112 @@
+package org.firstinspires.ftc.team6220_2020;
+
+/**
+ Used to store important constants for easy access in other classes.
+ */
+
+public class Constants
+{
+    // Standard conversions------------------------------------------
+    // This is for an AndyMark 40; 20s are 560 ticks / rot and 60s are 1680 ticks / rot
+    public static final int AM_40_TICKS_PER_ROTATION = 1120;
+    public static final int US_DIGITAL_TICKS_PER_ROTATION = 1440;   // For odometry wheels
+    public static final double MM_PER_INCH = 25.4;
+    public static final double IN_FIELD_SIZE = 140.94;  // Not 144" due to interlocking pieces being cut off on field edge.
+    //---------------------------------------------------------------
+
+
+    // Tolerances----------------------------------------------------
+    public static final double ANGLE_TOLERANCE_DEG =  1.0;
+    public static final double POSITION_TOLERANCE_IN = 0.5;
+    public static final double OPENCV_TOLERANCE_PIX = 0.5;
+    public static final double LIFT_MOTOR_TOLERANCE_ENC_TICKS = 30; // Tolerance for automatic lift movements
+    //---------------------------------------------------------------
+
+
+    // Robot specs---------------------------------------------------
+    public static final double WHEEL_DIAMETER_IN = 4.0;         // 4 inch diameter wheel
+    public static final double SPROCKET_RATIO = 16.0 / 16.0;    // Driven to driving sprocket teeth
+    public static final double IN_PER_ANDYMARK_TICK = (Math.PI * WHEEL_DIAMETER_IN) / (0.5 * AM_40_TICKS_PER_ROTATION * SPROCKET_RATIO);    // Num inches per tick of drive motors
+    //---------------------------------------------------------------
+
+
+    // Drive mode constants------------------------------------------
+    public static final double SLOW_MODE_T_FACTOR = 0.3;        // Slow mode multipliers
+    public static final double SLOW_MODE_R_FACTOR = 0.3;
+    public static final double T_FACTOR = 1.0;                  // Normal multipliers
+    public static final double R_FACTOR = 1.0;
```

Comment:
What are T and R factor?

Reply:
I don't know so I can remove them

Reply:
Wait, those are the numbers we use to set the max speed in slow mode and full speed mode.